HIP/hip-906.md

```diff
@@ -0,0 +1,125 @@
+---
+hip: 906
+title: Proxy Redirect Contract for Hbar Allowance and Approval
+author: Luke Lee <@lukelee-sl>
+working-group: Nana Essilfie-Conduah <@nana-ec>
+requested-by: EVM developers
+type: Standards Track 
+category: Service
+needs-council-approval: Yes
+status: Review
+created: 2024-02-23
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/906
+updated: 2024-03-06
+requires: 632
+---
+
+## Abstract
+
+The Smart Contract service currently provides functionality to grant allowance and approval for tokens from an owner account to a spender account. 
+However, there is currently no way to grant allowance and approval for hbars from an owner account using the Smart Contract service. This HIP proposes to remedy this omission.
+
+## Motivation
+
+Smart Contracts developers face an obstacle when implementing certain use cases for transferring hbars between accounts and contracts. 
+The lack of a mechanism to grant allowance and approval for hbars without using HAPI hinders developers’ workflow. Providing this functionality will enhance the developer experience and remove this limitation.
+
+## Rationale
+
+This HIP proposes adding the missing functionality via a new interface called `IHRC632`, which will act on an account address. 
```

Comment:
Might want to call out that it's building on the IHRC contract interface from HIP 632, hence the number